proc/proc_windows.go

```diff
@@ -0,0 +1,423 @@
+package proc
+
+// #include "windows.h"
+import "C"
+import (
+	"debug/gosym"
+	"debug/pe"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"sync"
+	"syscall"
+	"unsafe"
+
+	sys "golang.org/x/sys/windows"
+
+	"github.com/derekparker/delve/dwarf/frame"
+	"github.com/derekparker/delve/dwarf/line"
+)
+
+const (
+	// DEBUGONLYTHISPROCESS tracks https://msdn.microsoft.com/en-us/library/windows/desktop/ms684863(v=vs.85).aspx
+	DEBUGONLYTHISPROCESS = 0x00000002
+)
+
+// OSProcessDetails holds Windows specific information.
+type OSProcessDetails struct {
+	hProcess    sys.Handle
+	breakThread int
+}
+
+// Launch creates and begins debugging a new process.
+func Launch(cmd []string) (*Process, error) {
+	argv0Go, err := filepath.Abs(cmd[0])
+	if err != nil {
+		return nil, err
+	}
+	// Make sure the binary exists.
+	if filepath.Base(cmd[0]) == cmd[0] {
+		if _, err := exec.LookPath(cmd[0]); err != nil {
+			return nil, err
+		}
+	}
+	if _, err := os.Stat(argv0Go); err != nil {
+		return nil, err
+	}
+
+    argv0, _ := syscall.UTF16PtrFromString(argv0Go)
```

Comment:
Looks like there is some strange formatting in this function. Can you make sure gofmt has been run on all source files?